core/src/main/java/edu/bsu/storygame/core/view/NotebookLayer.java

```diff
@@ -345,7 +345,6 @@ private ReactionButton(final Reaction reaction) {
                         @Override
                         public void onEmit(Button button) {
                             context.reaction.update(reaction);
-                            context.story.update(reaction.stories.chooseOne());
```

Comment:
In case anyone is wondering, this call was made twice for some reason, and this was causing only one story to ever be displayed (there are a max of two stories per reaction). Removing this allowed both stories to be displayed.